### PR TITLE
Email customer when appointment is cancelled by them or PensionWise

### PR DESCRIPTION
**Pull request by AndrewVos**
~~Do not merge until @benbarnett has adjusted the content for this.~~

image

image

### CODE REVIEW

File: app/lib/notifier.rb [hunk 1]
```diff
@@ -23,9 +23,20 @@ def guiders_to_notify(appointment)
...
+  end
+
+  def appointment_time_changed?(appointment)
+    appointment.previous_changes.slice('start_at', 'end_at').present?
+  end
 
-    AppointmentMailer.updated(appointment).deliver_later
+  def appointment_cancelled?(appointment)
+    appointment.previous_changes.slice('status').present? &&
+      %w(cancelled_by_customer cancelled_by_pension_wise).include?(appointment.status)
```
Comment (benlovell) [inline app/lib/notifier.rb]: `Appointment#cancelled?` is a thing.

File: app/views/appointment_mailer/cancelled.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
```
Comment (benlovell) [inline app/views/appointment_mailer/cancelled.html.erb]: 🔪

File: spec/features/agent_manages_appointments_spec.rb [hunk 3]
```diff
@@ -64,6 +64,30 @@
     end
   end
 
+  scenario 'Agent cancels an appointment' do
+    given_the_user_is_an_agent do
+      and_there_is_an_appointment
+      and_they_cancel_the_appointment
```
Comment (benlovell) [inline spec/features/agent_manages_appointments_spec.rb]: `when_they_cancel_the_appointment`

Comment (benlovell) [inline spec/features/agent_manages_appointments_spec.rb]: Same elsewhere.

**Comment by benbarnett** [comment]
Hold up let me just try and get the content for this so we can do it in one fell swoop

**Comment by AndrewVos** [comment]
@benbarnett Updated :eye:

**Pull request merged**